test/end-to-end/ui-test.ts

```diff
@@ -1,17 +1,16 @@
 import * as assert from "assert";
-// @ts-ignore
-import * as mocha from "mocha";
```

Comment:
Don't we need this line any longer? I remember it existing as a way for the vscode linter to not complain about the test directives like `describe` and `it`. Could you check if this line can really be removed?